ppml/trusted-big-data-ml/python/docker-gramine/kubernetes/bigdl-on-volcano-helm/bigdl-pyspark-ncf.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: bigdl-pyspark-ncf
+spec:
+  type: Python
+  pythonVersion: "3"
+  mode: cluster
+  image: "10.239.45.10/arda/intelanalytics/bigdl-k8s-spark-3.1.2:2.1.0-SNAPSHOT-20220509"
+  imagePullPolicy: Always
+  mainApplicationFile: local:///bigdl2.0/data/spark_pandas.py
+  arguments:
+   - -f
+   - /bigdl2.0/data/nyc_taxi.csv
+  deps:
+    pyFiles:
+     - local:///opt/bigdl-2.1.0-SNAPSHOT/python/bigdl-spark_3.1.2-2.1.0-SNAPSHOT-python-api.zip
+     - local:///bigdl2.0/data/transfer_learning.py
+  sparkVersion: "3.1.2"
+  batchScheduler: "volcano"
+  restartPolicy:
+    type: Never
+  sparkConf:
+    "spark.driver.host": "10.244.7.74"
+    "spark.driver.port": "10008"
+    "spark.cores.max": "64"
+    "spark.kubernetes.authenticate.driver.serviceAccountName": "spark"
+    "spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName": "nfsvolumeclaim"
+    "spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl2.0/data"
+    "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName": "nfsvolumeclaim"
+    "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl2.0/data"
+    "spark.archives": "file:///bigdl2.0/data/le/pandas-environment.tar.gz"
+    "spark.pyspark.driver.python": "/bigdl2.0/data/le/python_env/bin/python"
```

Comment:
remove `/le` here? We'd better use a more general path here, like `/bigdl2.0/data` when open-source the deployment.

Reply:
also check the `driver.host` and `http_proxy`? we cannot push this kind of information directly in a public repo 😂.

Reply:
Going to refine this draft and replace them with placeholders from values.yaml later.